build.sbt

```diff
@@ -141,7 +141,7 @@ onLoad.in(Global) ~= { old =>
       """#!/bin/sh
         |set -eux
         |bin/scalafmt --diff
```

Comment:
I think this needs “-diff-branch=main”

Reply:
ah, whoops, I put in on the line below with `git diff` 😬 

I'll switch this.